src/dotnet-toolbox/UninstallCommand.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading.Tasks;
+using DotNetToolbox.Helpers;
+using Microsoft.Extensions.CommandLineUtils;
+using Newtonsoft.Json;
+
+namespace DotNetToolbox
+{
+    public class UninstallCommand : CommandLineApplication
+    {
+        private ToolboxConfiguration _toolboxConfig;
+        public UninstallCommand(CommandLineApplication parent)
+        {
+            Parent = parent;
+            Name = "uninstall";
+            PackageArgument = new PackageArgument(this);
+            Arguments.Add(PackageArgument);
```

Comment:
True, but it would not be synced with the `install` operation where you give a package ID.

If the problem is finding out the binary name, then we could do that on install, write it to the install file and then on uninstall just query the versions file and remove the binary and remove the entry in the versions file.

Thoughts?